templates/Docker/express/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# An official Node.js runtime (Alpine version) as a parent image
+FROM node:20-alpine
+
+# Set the working directory inside the container - Customizable.
+WORKDIR /app
+
+# Copy package.json to the working directory.
+COPY package.json ./
+
+# Install dependencies, including nodemon for development.
+RUN npm install nodemon --save-dev && npm install
+
+# Copy the application code to the container.
+COPY . .
+
+# TODO: Get the Port to be Exposed from .env => Can be Done after .env.local template is Ready.
+# Expose the port the app runs on - Customizable.
+EXPOSE 8080
+
+# Command to start the application with nodemon - Customizable.
+CMD ["npx", "nodemon", "app.js"]
```

Comment:
Default need not be with `nodemon`. Just starting it works.
Somebody who's new to this, say they directly deploy using the Dockerfile we generate. `nodemon` will be an overhead in that case for sure